packages/keplr/src/signer.ts

```diff
@@ -0,0 +1,246 @@
+import { AccountData, DirectSignResponse } from "@cosmjs/proto-signing";
+import {
+  ChainInfo,
+  Keplr,
+  Window as KeplrWindow,
+  KeplrSignOptions,
+} from "@keplr-wallet/types";
+import { SignDoc } from "cosmjs-types/cosmos/tx/v1beta1/tx";
+import { AminoSignResponse, StdSignDoc, Algo } from "@cosmjs/amino";
+import { assert } from "@cosmjs/utils";
+import { Signer, SignerStatus, SigningMode } from "@desmoslabs/desmjs";
+
+declare global {
+  interface Window extends KeplrWindow {}
+}
+
+export interface KeplrSignerOptions {
+  signingMode: SigningMode;
+  chainInfo: ChainInfo;
+  signOptions: KeplrSignOptions;
+}
+
+export const DesmosMainnet: ChainInfo = {
+  chainId: "desmos-mainnet",
+  chainName: "Desmos",
+  rpc: "https://rpc.mainnet.desmos.network",
+  rest: "https://api.mainnet.desmos.network",
+  bip44: {
+    coinType: 852,
+  },
+  bech32Config: {
+    bech32PrefixAccAddr: "desmos",
+    bech32PrefixAccPub: "desmospub",
+    bech32PrefixValAddr: "desmosvaloper",
+    bech32PrefixValPub: "desmosvaloperpub",
+    bech32PrefixConsAddr: "desmosvalcons",
+    bech32PrefixConsPub: "desmosvalconspub",
+  },
```

Comment:
Since these are the same between mainnet and testnet, can we somehow extract them so we don't have to replicate them? Also I think it would be better to move these constants into another file (maybe named `constants.ts` or something)